apps/web/.env.example

```
@@ -13,7 +13,7 @@ NEXT_PUBLIC_USE_FIREBASE_EMULATORS=false
 NEXT_PUBLIC_FIREBASE_EMULATOR_PROJECT_ID=demo-trr
 NEXT_PUBLIC_REALITEASE_FEEDBACK_URL=
 NEXT_PUBLIC_BRAVODLE_FEEDBACK_URL=
-NEXT_PUBLIC_HOSTED_FONT_BASE_URL=https://pub-a3c452f3df0d40319f7c585253a4776c.r2.dev
+NEXT_PUBLIC_HOSTED_FONT_BASE_URL=https://media.thereality.report
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_HOSTED_FONT_BASE_URL key should go before the NEXT_PUBLIC_REALITEASE_FEEDBACK_URL key

(UnorderedKey)